docs/Contributing/API-for-contributors.md

```diff
@@ -2692,5 +2695,32 @@ If the Fleet instance is provided required parameters to complete setup.
 
 ```
 
+## Scripts
+
+### Batch-apply scripts 
+
+_Available in Fleet Premium_
+
+`POST /api/v1/fleet/scripts/batch`
+
+#### Parameters
+
+| Name      | Type   | In    | Description                                                                                                                                                           |
+| --------- | ------ | ----- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| team_id   | number | query | The team ID to add the scripts to. Only one of team_name/team_id can be provided.                                                                                     |
+| team_name | string | query | The name of the team to add the scripts to. Only one of team_name/team_id can be provided.                                                                            |
+| dry_run   | bool   | query | Validate the provided scripts and return any validation errors, but do not apply the changes.                                                                         |
+| scripts   | array  | body  | An array of objects with the scripts payloads. Each item must contain `name` with the script name and `script_contents` with the script contents encoded in base64    |
+
+If no team (id or name) is provided, the scripts are applied for all hosts. After the call, the provided list of `scripts` will be the active scripts for that team (or no team) - that is, any existing script that is not part of that list will be removed, and an existing script with the same name as a new script will be edited. If the list of provided `scripts` is empty, all scripts are removed for that team (or no team).
```

Comment:
@roperzh I just realized this is the API for contributors and not the stable REST API, so I was probably being a bit too nitpicky with these changes 🤦‍♀️  Doesn't hurt to make them anyway, in case we ever bring this into the stable API.

(I'm going to request a similar set of changes to the REST API docs.)